contrib/builder/rpm/centos-6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
```

Comment:
though, it will as soon as you commit the build it is for, then that changes ;-)